src/xdocs-test/resources/com/puppycrawl/tools/checkstyle/checks/design/visibilitymodifier/Example7.java

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
Same applies here
```
Violation lines for /home/stoyan/open-source/checkstyle/src/xdocs-test/resources/com/puppycrawl/tools/checkstyle/checks/design/visibilitymodifier/Example7.java differ.
unexpected (3): 39, 41, 45
---
expected      : [29, 31, 33, 43, 47, 50, 53]
but was       : [29, 31, 33, 39, 41, 43, 45, 47, 50, 53]
```

Reply:
Let's not fight with examples, they simply should how tool works now.

If we think that validation if not good, we can create separate issue, outside of summer project to resolve. 
Please reflect detected violation set in Examples.

Reply:
Done.